catkin_ws/src/cv/launch/cv.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+	<arg name="camera" />
+	<node name="cv" pkg="cv" type="detection.py">
+		<param name="camera" value="$(arg camera)" />
+	</node>
```

Comment:
Just realized that this method of using arg may not work, since nodes need to have different names. Four ways to accomplish this: 
1. Use anonymous nodes. See [this](https://wiki.ros.org/roslaunch/XML).
2. Rather than using this arg method, just create the node itself in each of other launch files (cv_down.launch ,etc). You would delete this file and not use args at all (but you would have params in those files).
3. In this launch file, use some sort of string composition adding the arg to the node name.
4. Pass in an arg that denotes the node name.

Reply:
Sorry haven't gotten around to making these changes yet, but hopefully we'll have them done within the next few meetings